gecoscc/tasks.py

```diff
@@ -770,18 +770,14 @@ def group_by_multiple_keys(self, input_data, mergeIdField, mergeActionField, opp
         keygetter = itemgetter(*mergeIdField)
         result = []
         for key, grp in groupby(sorted(input_data, key = keygetter), keygetter):
-           temp_dict = dict({mergeActionField:[]})
-           for d in grp:
-              aux = d.copy()
-              temp_dict[mergeActionField] += [aux.pop(mergeActionField)]
-              temp_dict.update(aux)
-           if opposite:
-              # Incoherence in node
-              # temp_dict[mergeActionField] = ['add','remove']
-              if len(temp_dict[mergeActionField]) > 1:
-                  continue
-           temp_dict[mergeActionField] = temp_dict[mergeActionField][-1] # input_data sorted (ous,groups,computers). Last item has the highest priority
-           result.append(temp_dict)
+            group = list(grp)
```

Comment:
Are these changes related to the PR?

Reply:
From now on, policies will have a different number of fields depending on the action (add/remove) that user choose in policies forms.

Reply:
That is an important change. We need to test this thoroughly